app/templates/base.html

```diff
@@ -12,11 +12,12 @@
                 <li><a href="about">About</a></li>
                 <li><a href="#">Profile</a></li>
                 <li><a href="navigation">Navigation</a></li>
+                <li><a href="#">Contact Us</a></li>
+                <li><a href="user_profile">User Profile</a></li>
```

Comment:
This needs to be enclosed in the Jinja templating style as the links/routes below it.